source/com.microsoft.tfs.client.clc/src/com/microsoft/tfs/client/clc/Application.java

```diff
@@ -147,6 +147,10 @@ public int run(final String[] args) {
          */
         ProductInformation.initialize(ProductName.CLC);
 
+        // We need to check for the disable telemetry flag before we send any
+        // telemetry to the server
+        TfsTelemetryHelper.checkDisableTelemetryProperty();
```

Comment:
Shouldn't we use the CLCTelemetryHelper class here and in two other calls in this method? (For consistency :-))